process-link/src/main/kotlin/com/ritense/processlink/service/ProcessLinkActivityService.kt

```diff
@@ -64,19 +74,63 @@ class ProcessLinkActivityService(
         val processLink = processLinkService.getProcessLinksByProcessDefinitionIdAndActivityType(
             processDefinitionId,
             ActivityTypeWithEventName.START_EVENT_START
-        ) ?: return null
+        ) ?: throw ProcessLinkNotFoundException("For process definition with id $processDefinitionId")
```

Comment:
I don't thing the FE knows how to react to this exception yet. The will no longer show the 'please configure a start form' -message